src/rules/no-unused-modules.js

```diff
@@ -17,18 +18,27 @@ import includes from 'array-includes'
 let listFilesToProcess
 try {
   var FileEnumerator = require('eslint/lib/cli-engine/file-enumerator').FileEnumerator
-  listFilesToProcess = function (src) {
-    var e = new FileEnumerator()
+  listFilesToProcess = function (src, extensions) {
+    var e = new FileEnumerator({
```

Comment:
weird that these are still `var`s, could we change them to const and let?

Reply:
I was surprised as well. I update these 2 usages of `var`, however, there are more of them in the repository. I would leave the rest as is in the scope of this PR.

Reply:
agreed, thanks